pkg/types/builder.go

```diff
@@ -32,6 +32,10 @@ import (
 	"github.com/crossplane-contrib/terrajet/pkg/types/comments"
 )
 
+const (
+	wildcard = "*"
```

Comment:
I remember that we were able to get rid of wildcard in our fieldpath notations since we require a function in for additional connection keys now. Do I remember correctly or do you think there are still places where we need the wildcard?

Reply:
We got rid of wildcards in the Terrajet configuration API but internally we are still using it to [expand fieldpaths](https://github.com/crossplane/crossplane-runtime/blob/master/pkg/fieldpath/paved.go#L136).

Reply:
This is independent of additional connection keys, e.g. to identify a sensitive field in Terraform schema which contains maps or lists.